feat(curriculum): add animations to block 3.2 of the English Curriculum (#54033)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-hobbies-and-interests/657d7a4ad41e438c5be0a138.md

title: Task 13
challengeType: 22
dashedName: task-13
---

<!-- (Audio) Bob: For nearly a decade now. It started as a hobby, but it has become a significant part of my life. -->

# --description--

> When you say you do something `as a hobby`, it means that you do this regularly to enjoyment yourself, neither professionally, nor as an obligation.

Reviewer: ```suggestion
When you say you do something `as a hobby`, it means that you do this regularly for enjoyment, not professionally or as an obligation.
```
